recirq/otoc/otoc_example.py

```diff
@@ -77,33 +90,28 @@ def main():
     for i, circuits_i in enumerate(circuit_list):
         results_i = np.zeros((num_qubits, len(cycles)))
         for c, circuits_ic in enumerate(circuits_i):
-            print('Measuring circuit instance {}, cycle {}...'.format(i, c))
+            print("Measuring circuit instance {}, cycle {}...".format(i, c))
             stats = int(2000 + 10000 * (c / max(cycles)) ** 3)
-            sweep_params = [{} for _ in range(len(circuits_ic))]
-            job = engine.run_batch(
-                programs=circuits_ic,
-                params_list=sweep_params,
-                repetitions=stats,
-                processor_ids=[processor_name],
-                gate_set=cirq.google.FSIM_GATESET)
+            params = [{} for _ in range(len(circuits_ic))]
+            job = sampler.run_batch(programs=circuits_ic, params_list=params, repetitions=stats)
```

Comment:
What happens if you omit `params_list`? IIRC this was broken at some point but I thought it was fixed so you don't have to supply this argument if you aren't using params.

Reply:
This is the error message: "ValueError: Number of circuits and sweeps must match"

It is a bit odd since if I set the sampler to be `cirq.Simulator() `instead of a processor, it doesn't raise this error message even if I omit `param_list`. This also appears to be a Cirq version issue. I'm using Cirq 0.9.1 as listed in the requirements.txt. This problem doesn't exist if it is upgraded to 0.10.

I will raise an issue here to call for Cirq upgrade.

Reply:
ok cool, that makes sense